Makefile

```diff
@@ -12,7 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+.PHONY: push-multiarch-% push-multiarch
+
 CMDS=csi-node-driver-registrar
 all: build
 
 include release-tools/build.make
+
+# Override registry, revision and Images
+REV=$(REV_CI)
+REGISTRY_NAME=$(REGISTRY_CI)
+IMAGE_NAME=$(REGISTRY_NAME)/$*
```

Comment:
Can you do this without `REGISTRY_CI` and `REV_CI`? The goal is to write a make target that can be copied 1:1 into `release-tools/build.make`, with as few special cases as possible.

You can use `REV` and `REGISTRY_NAME` and override them with `make REV=... REGISTRY_NAME=...` in the `cloudbuild.yaml` invocation.

Reply:
Removed extra CI variables. Updated push-multiarch invocation as `make push-multiarch REV=v$(echo $_GIT_TAG | cut -f3- -d 'v') REGISTRY_NAME=gcr.io/k8s-staging-csi` in cloudbuild